bin/main.rb

```diff
@@ -1,2 +1,103 @@
 #!/usr/bin/env ruby
-puts 'Hello World'
\ No newline at end of file
+
+puts 'Welcome to Tic Tac Toe game.'
+puts '\n\n'
+puts '----------------Instructions------------'
+puts 'The game require two players'
+puts 'Player can choose number 1 to 9 to represent a move    '
+puts 'But a number(move) can only be played once'
+puts ''
+
+class Player
+  attr_accessor :player_x
+
+  attr_accessor :player_o
+
+  def initialize
+    # Collect names if the players
+    puts 'Player 1, Enter Name:'
+    @player_x = gets.chomp
+    puts 'Player 2, Enter Name:'
+    @player_o = gets.chomp
+    # Show the name of the player and the symbol
+    puts "#{@player_x} will be X AND #{@player_o} will be O"
+  end
+end
+
+class Board
+  attr_reader :board
+
+  # Show what the Tic Tac Toe Board look like
+  def initialize
+    puts 'This is the board of Tic Tac Toe '
+    puts '  ___ ___ ___  '
+    puts ' | 1 | 2 | 3 | '
+    puts ' |___|___|___| '
+    puts ' | 4 | 5 | 6 | '
+    puts ' |___|___|___| '
+    puts ' | 7 | 8 | 9 | '
+    puts ' |___|___|___| '
+    puts '               '
+
+    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+  end
+
+  # Display board after a player has played
+  def display_board(board)
+    puts "#{board[0]} | #{board[1]} | #{board[2]}"
+    puts '---------'
+    puts "#{board[3]} | #{board[4]} | #{board[5]}"
+    puts '---------'
+    puts "#{board[6]} | #{board[7]} | #{board[8]}"
+  end
+end
+
+class Game
+  # Start a new game by geting player names
+  def initialize
+    @player = Player.new
+    @board = Board.new
+  end
+
+  def play
+    game_over = false
+    while game_over
+      puts 'X, Enter 1-9 to play:'
+      player_x_move = gets.chomp
+      puts 'O, Enter 1-9 to play:'
+      player_o_move = gets.chomp
+
+      puts "x played #{player_x_move} And o played #{player_o_move}"
+
+      game_over = true if winner
+    end
+  end
```

Comment:
You have mapped out scenarios of how the game tells which player turn it is. Also within this loop, map out the following scenarios:

- what does it look like when a player inputs an invalid move? Also, using puts/ print, indicate the message the player gets for an invalid move.

- What does it look like when the game is drawn? Also, using puts/prints, indicate the message the players receive when the game ends in a draw.

- What does it look like when the game is won? Also,  using puts/prints, indicate the message the player receives when they win a game.